### ISSUE TITLE
Problem with Restyler Job #2858746

### ISSUE THREAD

**Issue by falgon** [restyled-io/restyler#197]
Hi there-

I'm having a problem with a Restyled Job

- **Restyled Job**: https://restyled.io/gh/falgon/repos/roki-web/jobs/2858746
- **GitHub PR**: falgon/roki-web#410

**What I expected to happen**:
No internal errors

**What happened instead**:
An internal error has occurred.
```
[error]    Exception:
contents/roki.diary/2023/02/18/camel-resort-glamping/foot_golf_image.jpg: hGetContents: invalid argument (invalid byte sequence) error={description:"internal error",exitCode:99,severity:"critical",tag:"unknown"}
```

**Configuration**:

```yaml
enabled: true
restylers:
  - name: stylish-haskell
    include:
      - "**/*.hs"
  - name: dhall-format
    include:
      - "**/*.dhall"
pull_requests: true
statuses:
  differences: true
  error: true
labels:
  - refactoring
  - automerge
ignore_labels:
  - restyled-ignore
request_review: author
```

**FYI**
For reference, the information of the jpeg file caught as an error is as follows.

```bash
$  file ./foot_golf_image.jpg && exiftool ./foot_golf_image.jpg
foot_golf_image.jpg: JPEG image data
ExifTool Version Number         : 10.80
File Name                       : foot_golf_image.jpg
Directory                       : .
File Size                       : 332 kB
File Modification Date/Time     : 2023:02:18 00:17:32+09:00
File Access Date/Time           : 2023:02:20 02:12:02+09:00
File Inode Change Date/Time     : 2023:02:18 00:17:32+09:00
File Permissions                : rwxrwxr-x
File Type                       : JPEG
File Type Extension             : jpg
MIME Type                       : image/jpeg
Image Width                     : 1040
Image Height                    : 1387
Encoding Process                : Baseline DCT, Huffman coding
Bits Per Sample                 : 8
Color Components                : 3
Y Cb Cr Sub Sampling            : YCbCr4:2:0 (2 2)
Image Size                      : 1040x1387
Megapixels                      : 1.4
```

**Comment by pbrisbin** [comment]
Hi, thanks for the report!

I'll admit, I was really stumped by this. I've hit and resolved this bug before, and we even have a passing test case for it. Unfortunately, the code didn't expect a binary image file to be _executable_, and that's what led to the erroring read.

```
File Permissions                : rwxrwxr-x
```

I'm pushing an update through now to handle this scenario correctly.

**Comment by pbrisbin** [comment]
I think this is fixed in bcd55b9b6d986002b7bb0c18f355240871fe3ee0. That should be deployed to prod in an hour or so.
